xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -649,9 +649,12 @@ namespace XBMCAddon
     /// @param id                    string - id of setting to return
     /// @return                      Region setting
     ///
-    /// @note choices are (dateshort, datelong, time, meridiem, tempunit, speedunit)
-    ///        You can use the above as keywords for arguments.
+    /// @note choices are (dateshort, datelong, time, meridiem, tempunit, speedunit,
+    ///       datelongraw, dateshortraw, timeraw)
+    ///       You can use the above as keywords for arguments.
     ///
+    ///       If returning an empty string id is not supported (e.g. older Kodi
```

Comment:
Maybe make this a warning and improve the wording since it reads confusing to me (also drop the Kodi version reference):
`/// @warning an empty string is returned if the provided Id is not supported`

After this I think it is good to go

Reply:
Is changed.